client-v2/src/shared/components/input/ButtonRoundedWithLabel.tsx

```diff
@@ -1,14 +1,6 @@
-import React, { ReactNode } from 'react';
 import { styled } from 'shared/constants/theme';
 
-interface ContainerButtonWithLabelProps {
-  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void;
-  label: string;
-  icon?: ReactNode | null;
-  className?: string;
-}
-
-const ContainerButton = styled('button')`
+export default styled.button`
```

Comment:
A brief tidy up here to make this component more composable and remove an unnecessary SFC.